package.json

```diff
@@ -4,7 +4,13 @@
   "description": "Action to handle automatic PR labeling",
   "main": "dist/index.js",
   "scripts": {
-    "build": "ncc build src/index.ts --out dist"
+    "test": "jest",
+    "build": "ncc build src/index.ts --out dist",
+    "prepare": "husky install"
+  },
+  "engines": {
+    "node": "12.x",
```

Comment:
But the actions only runs in `node12`, so I want to make it explicit.

Reply:
Why? This is weird limitation you're placing on the repo, and is also trying hard to prevent future testing the repo. With a test suite and such, we should be running the test suite against Node 12 and 16 minimally so that the repo is ready to jump to a new minimum version of Node when the main tldr-pages repo does. You can also force Node 12 compatibility through that test suite, like any other javascript project.

Reply:
I strongly recommend @MasterOdin's suggestion here. Expllicitly limiting it to Node.js 12.x would make this extremely difficult to e.g. run in a development environment. It would also lead to issues should GitHub Actions upgrade to a newer version of Node.js.